dbt/adapters/sqlserver/connections.py

```diff
@@ -358,16 +362,26 @@ def get_credentials(cls, credentials):
         return credentials
 
     @classmethod
-    def get_status(cls, cursor):
-        if cursor.rowcount == -1:
-            status = "OK"
-        else:
-            status = str(cursor.rowcount)
-        return status
+    def get_response(cls, cursor) -> AdapterResponse:
+        #message = str(cursor.statusmessage)
+        message = 'OK'
+        rows = cursor.rowcount
+        #status_message_parts = message.split() if message is not None else []
+        #status_messsage_strings = [
+        #    part
+        #    for part in status_message_parts
+        #    if not part.isdigit()
+        #]
+        #code = ' '.join(status_messsage_strings)
+        return AdapterResponse(
+            _message=message,
+            #code=code,
+            rows_affected=rows
+        )
 
     def execute(self, sql, auto_begin=True, fetch=False):
         _, cursor = self.add_query(sql, auto_begin)
-        status = self.get_status(cursor)
+        status = self.get_response(cursor)
         if fetch:
             table = self.get_result_from_cursor(cursor)
         else:
```

Comment:
what's this about? a change in `dbt-core` I assume?

Reply:
yes! (https://github.com/fishtown-analytics/dbt/pull/2961)